src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall(Lottery ball) {
+        int number = new Random().nextInt(100);
+        return "Ball has number: " + number + " and color: " + ball.getRandomColor();
+    }
```

Comment:
Avoid using inheritance where it is not necessary. Here, the Lottery class should not extend ColorSupplier. Instead, it should use an instance of ColorSupplier.